add associations (#535)
akeneo/pim-helpcenter: content/md/productivity/get-started-with-the-rules-engine.md

To set the *“My very new description for purple T-shirt”* value to your `description` attribute in the `en_US` locale and for the `e-commerce` channel, the action will be as follows:

For instance, the following actions will **disable** the product and set its **family** to `shoes`. It will also **categorize** it in `casual` and `women` (while uncategorizing it from its previous categories), and **add** it to the `summer` group (while removing it from its previous groups).

```YML
  actions:
      - type: set
        field: enabled
        value: false
      - type: set
        field: family
        value: shoes
      - type: set
        field: categories
        value:
           - casual
           - women

>       - type: groups

Reviewer: ```suggestion
      - type: set
```

        field: groups
        value:
          - summer
```

#### Set associations

> As for the add action, you can choose to associate any combination of **products**, **product_models** or **groups** for each association type.

Reviewer 2: ```suggestion
Just like for the add action, you can choose to associate any combination of **products**, **product_models** or **groups** for each association type.
```

> 

Reviewer 2: For each association type, you can now decided which associated product, product_model or groups you want to update. The other ones will not be updated.

> For instance, the following action will replace the associated products for X_SELL, but won't update associated product models or groups.

Reviewer 2: ```suggestion
In the example below, you can see that the following action will replace the associated products for X_SELL, but won't replace associated product models or groups.
```

> On the opposite, it will replace its associated product models and groups for UPSELL association, but won't update associated products.

Reviewer 2: I'm not sure here...

Reviewer 2: ```suggestion
And for the UPSELL association, it will replace the associated product models and groups but not the associated products.
```
